proposals/exception-handling/Exceptions.md

```diff
@@ -499,10 +537,17 @@ throws, and rethrows as follows:
 | Name | Opcode | Immediates | Description |
 | ---- | ---- | ---- | ---- |
 | `try` | `0x06` | sig : `blocktype` | begins a block which can handle thrown exceptions |
-| `catch` | `0x07` | | begins the catch block of the try block |
+| `catch` | `0x07` | index : `varint32` | begins the catch block of the try block |
+| `catch_all` | `0x19` | | begins the catch_all block of the try block |
+| `unwind` | `0x0a` | | begins the unwind block of the try block |
+| `delegate` | `0x18` | relative_depth : `varuint32` | begins the delegate block of the try block |
 | `throw` | `0x08` | index : `varint32` | Creates an exception defined by the exception `index`and then throws it |
-| `rethrow` | `0x09` | | Pops the `exnref` on top of the stack and throws it |
-| `br_on_exn` | `0x0a` |  relative_depth : `varuint32`, index : `varuint32` | Branches to the given label and extracts data within `exnref` on top of stack if it was created using the corresponding exception `index` |
+| `rethrow` | `0x09` | relative_depth : `varuint32` | Pops the `exnref` on top of the stack and throws it |
 
 The *sig* fields of `block`, `if`, and `try` operators are block signatures
 which describe their use of the operand stack.
+
+Note that the textual `catch_all` instruction is implemented using the
+`else` operator. Since the `else` operator is always unambiguous in the binary
+format, there is no need to tie up a separate opcode for the `catch_all`
+instruction.
```

Comment:
I think this text about `catch_all` can be deleted now that it's assigned opcode `0x19`.

Reply:
Good catch, thanks. Removed.